client/landing/stepper/declarative-flow/internals/steps-repository/site-migration-identify/style.scss

```diff
@@ -83,6 +84,11 @@
 	max-width: 368px;
 	padding: 1.5rem;
 
+	@media (max-width: $break-small) {
```

Comment:
This would ideally be a mobile first media query (i.e. it should be `min-width` and not `max-width`). 🙂

Reply:
Thanks for pointing out! Forgot this good practice in a hurry! I'll take note to update it in the next PR